specification/attestation/data-plane/Microsoft.Attestation/preview/2022-09-01-preview/attestation.json

```
@@ -467,22 +467,14 @@
         "tags": [
           "TcbBaseline"
         ],
-        "summary": "Retrieves the current set Tcb baselines or all available Tcb baselines for an attestation type.",
+        "summary": "Retrieves a list of all azure supported baseline details for the a TEE type along with an indication of which one is in enforced for the attestation provider.",
```

We should properly case our own product names.
```suggestion
        "summary": "Retrieves a list of all Azure supported baseline details for the a TEE type along with an indication of which one is in enforced for the attestation provider.",
```

---

Several other instances below.

---

Thanks! Updated